.env.prod

```diff
@@ -0,0 +1,9 @@
+# Hare needs this
+HOST=0.0.0.0
+
+# Might be renamed to ENDPOINT_BASEURL
+LB_ADDR=http://example.private.local:8080
```

Comment:
What I mean is that LB_ADDR might be better renamed to BASEURL.
In other projects, I see the convention of ADDR as an IPv4/IPv6 address, not a string that starts by http and has no trailing slash.

Also, maybe it should be mentioning about the fact that this endpoint SHOULD BE from a **private** network, only accessible from Nuxt (i.e. not publicly exposed).

Opinion @clarkdo ?